Ryanyae 133 add member

ryanyae opened the pull request:
## Description

You are able to add new members to the DB.
## Updates
Project page updates

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

ryanyae commented:
![](https://github.trello.services/images/mini-trello-icon.png) [add member functionality](https://trello.com/c/wHk0AZ7k/133-add-member-functionality)

ElisaHu1 commented on package.json:
```diff
@@ -38,7 +38,7 @@
     "yup": "^0.32.9"
   },
   "scripts": {
-    "start": "export SET NODE_OPTIONS=--openssl-legacy-provider && react-app-rewired start",
```
this need to be reverted

ryanyae replied:
There is some bug with the .gitignore with my PC vs. my laptop. My laptop is fine, but my PC always gives me this issue. But yes, I will revert this!

ElisaHu1 reviewed (commented):
are you able to add comments in the code related to this feature?
What you have tested and what is the expected result

ryanyae commented:
> are you able to add comments in the code related to this feature? What you have tested and what is the expected result

Right, good idea. Noticed that I need to add more comments cause sometimes I go back on my own code and misunderstand what is happening.

ElisaHu1 commented:
merge conflict need to be resolved

ryanyae commented:
@ElisaHu1 Fixed the merge conflicts, fixed the api call (i was calling to localhost instead of the ec2), added a feature where if the call is successful the add member tab is auto close.

ElisaHu1 reviewed (approved):
looks good to me, but can you revert the package.json back plz?

ryanyae commented:
@ElisaHu1 Just reverted package.json file
